ansible_collections/arista/avd/roles/eos_designs/doc/ptp.md

```diff
@@ -131,11 +135,12 @@ All other node_types will have a default PTP priority 1 of `127` to ensure they
 For leaf switches connecting to a PTP GrandMaster we recommend to manually set PTP priority 1 lower than the other leaf switches, for example to "10" as shown below:
 
 ```yml
-nodes:
-  ptp-leaf1:
-    ptp:
-      enabled: true
-      priority1: 10
+l3leaf:
+  node_groups:
```

Comment:
You missed some levels from my example. Either create the full node_groups + nodes hierarchy, or change this key to `nodes`

Reply:
Fixed